libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/utils/ViewUtils.java

```diff
@@ -48,8 +49,12 @@ public static Bitmap loadBitmapFromView(View view) {
     return null;
   }
 
-  public static float dpToPx(Context context, int dp) {
+  public static int[] buildRouteOverviewPadding(Context context) {
```

Comment:
This doesn't sound like a `View` utility. It's only used in `NavigationView`. IMO, objects and methods should live/be where their context is. From my experience utility classes become a drawer easily and we should treat them carefully.